cornflow-server/cornflow/commands/views.py

```diff
@@ -68,3 +41,108 @@ def register_views_command(external_app: str = None, verbose: bool = False):
             current_app.logger.info("No new endpoints to be registered")
 
     return True
+
+
+def load_changes_to_db(views_to_delete, views_to_register, views_to_update):
+    if len(views_to_register) > 0:
+        db.session.bulk_save_objects(views_to_register)
+    if len(views_to_update) > 0:
+        db.session.bulk_update_mappings(ViewModel, views_to_update)
+    if len(views_to_delete) > 0:
+        for view_id in views_to_delete:
+            view_to_delete = ViewModel.get_one_object(idx=view_id)
+            if view_to_delete:
+                view_to_delete.delete()
+    try:
+        db.session.commit()
+    except IntegrityError as e:
+        db.session.rollback()
+        current_app.logger.error(f"Integrity error on views register: {e}")
+    except DBAPIError as e:
+        db.session.rollback()
+        current_app.logger.error(f"Unknow error on views register: {e}")
+
+
+def get_views_to_update_and_delete(
```

Comment:
I think it might be better to just make a different function for viers_to update and views_to_delete.

Also views_to_delete code is pretty easy:
[view_attrs["id"] for view_name, view_attrs in views_registered_urls_all_attributes.items() if view_name not in all_resources_to_register_views_endpoints.keys()

Reply:
and both arguments could have a simpler name

Reply:
for loops with append are less efficient than list comprehensions, so separating it might actually be faster